.github/workflows/ci-actions-incremental.yml

```
@@ -621,6 +628,81 @@ jobs:
             LICENSE.txt
           retention-days: 2
 
+  kubernetes-tests:
+    name: Kubernetes Tests - JDK ${{matrix.java.name}}
+    runs-on: ${{matrix.java.os-name}}
+    needs: [build-jdk11, calculate-test-jobs]
+    # Skip main in forks
+    if: "needs.calculate-test-jobs.outputs.run_kubernetes == 'true' && (github.repository == 'quarkusio/quarkus' || !endsWith(github.ref, '/main'))"
+    timeout-minutes: 40
+    strategy:
+      fail-fast: false
+      matrix:
+        java:
+          - {
+            name: "11",
+            java-version: 11,
+            os-name: "ubuntu-latest"
+          }
+          - {
+            name: "17",
+            java-version: 17,
+            os-name: "ubuntu-latest"
+          }
+          - {
+            name: "11 Windows",
+            java-version: 11,
+            os-name: "windows-latest"
+          }
+    steps:
+      - name: Support longpaths on Windows
+        if: "startsWith(matrix.java.os-name, 'windows')"
+        run: git config --global core.longpaths true
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v3
        with:
          fetch-depth: 0
```
See: https://github.com/quarkusio/quarkus/blob/3.3.0/.github/workflows/ci-actions-incremental.yml#L284-L285

This should resolve the "main" branch issue.

---

...and we'll also need
```
      - name: Add quarkusio remote
        shell: bash
        run: git remote show quarkusio &> /dev/null || git remote add quarkusio https://github.com/quarkusio/quarkus.git
```
because of https://github.com/quarkusio/quarkus/blob/3.3.0/.github/workflows/ci-actions-incremental.yml#L151-L154

---

This one I already added, didn't I?

---

https://github.com/quarkusio/quarkus/pull/35442/files#diff-d8e36253822cffbdf21e333bcba467345b46e6095927dda98399fe7f21f4679aR417-R419

---

yeah right - sorry for the noise, it's been a long week

---

No worries, same here :)

---

It’s still failing. I thing I will just go with running both for now until you can get a closer look.